src/Pagination/index.js

```diff
@@ -48,7 +49,7 @@ export const _getAttributes = (
 
   return {
     visiblePages,
-    selectedIndex,
+    selectedIndex: selectedIndex >= 0 ? selectedIndex : 0,
```

Comment:
`selectedIndex` here refers to the index of visible page numbers (the five we show in the middle).

When the selected page is out of bounds of the visible pages, we can assume it's also out of bounds of total pages and default the selection to index `0` (page 1).